### PR TITLE
Mention this guide requires only 1 master node

### PR DESCRIPTION
**Pull request by nirvanagit**

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
@nirvanagit  could you please ensure that the update note is in the Before you begin section.
https://deploy-preview-8376--kubernetes-io-master-staging.netlify.com/docs/setup/independent/create-cluster-kubeadm/

Right now the note appears at the end of the page.

**Comment by nirvanagit** [comment]
@Rajakavitha1 : Could not find `Before you begin section`

**Comment by Rajakavitha1** [comment]
@nirvanagit Before you begin section is here in this file: https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/

**Comment by mistyhacks** [comment]
Please address the feedback from @Rajakavitha1. If you are using the Github UI, browse to your fork, select the `patch-6` branch, and edit the file `/content/en/docs/setup/independent/create-cluster-kubeadm.md`. The If you have `patch-6` checked out when you do that, the commit will be added to this PR automatically.

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ timeframe; which also applies to `kubeadm`.
    private network is fine)
 {{% /capture %}}
 
+**Note:** This is not a [HA setup](https://kubernetes.io/docs/setup/independent/high-availability/), and requires only 1 master node.
```
Comment (luxas) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: I would say something more along the lines of 
```
**Note:** This getting started guide gives you a Kubernetes cluster with one master and a number of nodes of your choice, and hence requires only one master node by default.
If you want to follow an more advanced path and create a cluster with multiple masters, in order to achieve high availability, follow [this guide instead](https://kubernetes.io/docs/setup/independent/high-availability/)
```

**Comment by zacharysarah** [comment]
@nirvanagit Any updates?

**Comment by mistyhacks** [comment]
I'll carry this.

**Comment by mistyhacks** [comment]
The note is now reworded and appears in the right place.

/lgtm
/approve

**Pull request merged**